products/compute/terraform.yaml

```diff
@@ -1119,11 +1119,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           group_name: "soletenant-group"
           template_name: "soletenant-tmpl"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "node_group_autoscaling_policy"
+        min_version: beta
+        primary_resource_id: "nodes"
+        vars:
+          group_name: "soletenant-group"
+          template_name: "soletenant-tmpl"
     properties:
       zone: !ruby/object:Overrides::Terraform::PropertyOverride
         required: false
         default_from_api: true
         custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+      autoscalingPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Do these need to all be O+C? Is the `autoscalingPolicy` block returned from the API even if it is undefined?

Reply:
Yeah the block was returning from the API in the basic test without `autoscalingPolicy` being set in the config, and was throwing a diff.

If the block IS defined, `mode` and `maxNodes` are required.